web/server/entrypoint_linux.go

```diff
@@ -1,17 +1,23 @@
-//go:build cgo && linux && !android
+//go:build cgo && linux && !arm
 
 package server
 
 import (
 	"go.viam.com/rdk/gostream"
+	"go.viam.com/rdk/gostream/codec/h264"
 	"go.viam.com/rdk/gostream/codec/opus"
 	"go.viam.com/rdk/gostream/codec/x264"
+	"go.viam.com/rdk/gostream/ffmpeg/avcodec"
 )
 
 func makeStreamConfig() gostream.StreamConfig {
 	var streamConfig gostream.StreamConfig
-	// TODO(RSDK-5916): support v4l2m2m codec on arm64/rpi
-	streamConfig.VideoEncoderFactory = x264.NewEncoderFactory()
+	if avcodec.EncoderIsAvailable(h264.V4l2m2m) {
```

Comment:
This adds back the RPi HW encoder, but I left that ticket open and added it to the epic.

Reply:
`EncoderIsAvailable` check is nice and simple! 

[q] Are we sure we want to add back in without addressing edge case issues first?